pennylane/ops/qubit/observables.py

```diff
@@ -427,7 +427,7 @@ def __init__(self, state, wires, id=None):
 
         super().__init__(state, wires=wires, id=id)
 
-    def __new__(cls):  # pylint: disable=arguments-differ
+    def __new__(cls, *_, **__):  # pylint: disable=arguments-differ
```

Comment:
I still don't understand the `__new__` dunder very well. What is the point of changing the signature?

Reply:
These were failing pickling.  Basically, creating new instances wasn't happening correctly.  `__new__` needs to have the same signature as `__init__`.